growth/too/tasks/obs.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from astropy import time
+import astropy.units as u
+from pyvo.dal import TAPService
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from . import celery
+from .. import models
+
+log = get_task_logger(__name__)
+
+__all__ = ('ztf_obs',)
+
+
+def get_tap_client():
```

Comment:
This is duplicated from #18. After #18 is merged, could we combine these into a single module so that we can increase code reuse?

Reply:
Can you merge #18 so I can do this?